### DISCUSSION TITLE
双向认证过程

### DISCUSSION THREAD

**Discussion by ValorBao** [Tencent/tquic#130] [Q&A (问题和回答)] [answered]
收到服务器的server_hello,但是没有收到服务器下一个handshake包，当收到服务器的server_hello,需要做什么处理吗？

**Comment by xiaofei0800** [comment, marked as answer]
使用时不需要关注具体的握手报文。收到服务端报文后，通过quic_endpoint_recv将报文递交给TQUIC协议栈，然后通过quic_endpoint_process_connections处理连接事件，可参考[C示例客户端程序](https://github.com/tquic-group/tquic-example-c/blob/0c582a57aeca1fa220cf4535f41efa074fcaf1ff/simple_client.c#L167)中的process_connections函数。
